Catappult/appcoins-wallet-android
# Map error messages

**What does this PR do?**

Map error messages
Prevent error in the gas inputs
Adapt UI to the new error messages

**How should this be manually tested?**

To test this ticket you must:

Repeat the same transaction with the same NFT and should view the error message “This NFT is already being sent! A previous transaction is currently in progress.“

Make a transaction from an account with no ETH and receive the message “Not enough funds! You don't have enough ETH to pay for the fees.“

Preform a transaction and put a low value on the gas limit and should have the message “Gas is too low! Increase your gas for the network to accept your transaction.“

Every other action that result on an error should return the message “Oops! Something went wrong!“

The app should not crash or show an error when the gas input is empty

The error messages should not overflow the UI

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2980](https://aptoide.atlassian.net/browse/APPC-2980)

**Code Review Checklist**
